Remotion/ObjectBinding/Web.IntegrationTests/BocList/AccessibilityBocListTest.cs

```diff
@@ -0,0 +1,121 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests;
+using Remotion.Web.Development.WebTesting.Accessibility;
+using Remotion.Web.Development.WebTesting.ExecutionEngine.PageObjects;
+using Remotion.Web.Development.WebTesting.FluentControlSelection;
+
+namespace Remotion.ObjectBinding.Web.IntegrationTests.BocList
+{
+  [TestFixture]
+  public class AccessibilityBocListTest : IntegrationTest
+  {
+    [Test]
+    public void Normal ()
```

Comment:
Empty-test is needed in case the the markup is different. Does the Empty test use the placeholder text or just doesn't show data-rows? We should have tests for both versions.

Reply:
Nice catch ⚾ The `Empyt` BocList on our test site is the one with placeholder text. I've added another one without the placeholder and sure enough, it fails the "AriaRequiredChildren" rule